src/EFTools/EntityDesign/VisualStudio/ModelWizard/gui/WizardPageDbConfig.cs

```diff
@@ -676,8 +676,17 @@ private void UpdateSettingsFromGui()
 
             // these connection strings & invariant names are coming from the ddex provider, so these are "design-time"
             var invariantName = DataConnectionUtils.GetProviderInvariantName(_dataProviderManager, _dataConnection.Provider);
+            var fixedDecryptedConnectionString = ReplaceMdsKeywords(decryptedConnectionString);
```

Comment:
Wasn't this already changed in #41?

Reply:
No, it was not. That was a separate instance of the same issue